apiserver/facades/agent/unitassigner/unitassigner.go

```diff
@@ -113,7 +114,7 @@ func (a *API) AssignUnits(ctx context.Context, args params.Entities) (params.Err
 func (a *API) saveMachineInfo(ctx context.Context, machineId string) error {
```

Comment:
Sorry; I should have got this on the last review.

_We_ know that the (old) machine ID is congruent with the (new) machine name, but for people reading this code, the ID/name coercion will be a potential source of confusion.

In all these places where we are accepting the argument, call it name instead of ID. You obviously won't be able to do this for RPC params, which remain ID, but where we are making new declarations, it should be consistent.